### ISSUE TITLE
Publish

### ISSUE THREAD

**Issue by hitnrun30** [Chrisae9/phase10randomizer#1]
How do you publish this to a site to use for our own purposes.  I think I figured out the math to use instead of the json reference file.  I want to publish so I dont have to wait for you to put it on your dev system.

I am also working on ordering by easiest to hardest but the math is a bit beyond me.

**Comment by chrisae9** [comment]
Hello there! Thank you for taking the time to look at my repo. 🥇

I used a generic gatsby starter, the REAME.md in this repo should explain how to build and deploy a gatsby starter.
https://github.com/taylorbryant/gatsby-starter-tailwind

For my deployment, I used my own personal server, but you should be able to run it using github-pages.
https://pages.github.com/

Feel free to fork the code and create a pull request :)

**Comment by hitnrun30** [comment]
I am going to publish in a bit with a fork, but I wanted it on my own server to use and test

also as an FYI unless there was a reason to use gatsby(have no idea what that is)
https://www.delftstack.com/howto/python/python-in-html/

**Comment by hitnrun30** [comment]
I also am new to the gatsby and such.  I got the site up but if I'm reading it right, its not using the phase10logic.py.  its just using the data, randomizing the phase numbers then randomly selecting one of the entries.  How would I change the below to actually reference what is spit out from phase10logic.py?

import phase10data from "../data/phase10data.json";

  var chosen = [];
  var selection = "";
  var phasenumbers = ["1","2","3","4","5","6","7","8","9","10"].sort(() => Math.random() - 0.5)

  phasenumbers.map(function(phase){
    while (true) {
      selection = phase10data[phase][Math.floor(Math.random() * phase10data[phase].length)];
      if (!chosen.includes(selection)) {
        chosen.push(selection)
        break;
      }
    }
  })

**Comment by chrisae9** [comment]
You are correct, I only intended the Python file to be a proof of concept before I created the website.

I originally tried to import the Python code to be used in the website but I couldn't find a solution quicker than just redoing the code in JavaScript.

This might be helpful:
https://stackoverflow.com/questions/13175510/call-python-function-from-javascript-code

Seems like it is possible to call a python file using JQuery.

Hopefully this helps!
